.github/workflows/inputs-remote-copy.yml

```diff
@@ -0,0 +1,90 @@
+name: Config Inputs Remote Copy
+run-name: "${{ inputs.remote-environment }} Config Input Copy to ${{ inputs.target }}"
+on:
+  workflow_dispatch:
+    inputs:
+      remote-environment:
+        type: choice
+        required: true
+        description: The Github Environment for the given remote
+        options:
+          - Gadi
+      source:
+        type: string
+        required: true
+        description: Remote absolute path to configuration input file/folder
```

Comment:
We need to be quite careful with trailing slashes in rsync

https://stackoverflow.com/a/31278462

It is *really* easy to not do what you expected.

Reply:
This is true. But I think that it should be a responsibility of the user to write their `inputs.source` correctly rather than the CI to modify the command that they have explicitly written. What do you think?

Reply:
Depends what this workflow is designed for. We could make not use `rsync` at all and be very prescriptive (and helpful!) and automate versions etc and be designed to just put new files in a single location to update an existing large directory tree.

Then we could have a different, related, workflow that is really supposed to copy whole directory trees, but it would happen rarely and maybe would definitely *not* overwrite any existing files.

If we don't do above we should add some warnings in the description/docs to be mindful of trailing slashes!

Reply:
I think for now we should go with the "warning the user" bit. We can iterate upon this idea in the future, I think.